manifest.yml

```diff
@@ -1,8 +1,8 @@
 ---
 applications:
   - name: vault-broker
-    command: cf-vault-service-broker
-    buildpack: https://github.com/cloudfoundry/go-buildpack.git#v1.8.33
+    command: vault-service-broker
+    buildpack: https://github.com/cloudfoundry/go-buildpack.git#v1.9.48
     env:
-      GOVERSION: go1.11.5
-      GOPACKAGENAME: github.com/hashicorp/cf-vault-service-broker
+      GOVERSION: go1.18.3
```

Comment:
1.18.5 to match above?

Reply:
I think it'll have to be 1.18.3, since that's the highest version the buildpack supports: https://github.com/cloudfoundry/go-buildpack/releases/tag/v1.9.48